doc/controls/SafeArea.md

```diff
@@ -34,6 +34,23 @@ xmlns:utu="using:Uno.Toolkit.UI"
 </SafeArea>
 ```
 
+> [!WARNING]
+> **Always wrap form inputs with SafeArea when using TextBox or PasswordBox**
+>
+> On mobile devices, the on-screen keyboard will obscure input fields unless you wrap them with `SafeArea` using `Insets="SoftInput"` or `Insets="SoftInput,Bottom"`. This is especially critical for login forms and any page with text entry controls.
+>
+> ```xml
+> <utu:SafeArea Insets="SoftInput,Bottom">
+>     <ScrollViewer>
```

Comment:
We don't need a ScrollViewer here, it should be automatically injected by the SafeArea control since the Insets contain `SoftInput`. See the current docs just below this code block